pyproject.toml

```
@@ -6,9 +6,13 @@ readme = "README.md"
 authors = [
     { name = "Johann Schopplich", email = "hello@johannschopplich.com" }
 ]
-requires-python = ">=3.8"
+requires-python = ">=3.10"
 dependencies = [
```

These should be in [dependency-groups.dev]

- Runtime deps should be minimal (dev tool, not runtime dependency)
```suggestion
[dependency-groups]
dev = [
    "mypy>=1.18.2",
    "pytest>=9.0.0",
    "ruff>=0.14.4",
    "tox>=4.25.0",
    "tox-uv>=1.13.1",
]
```